src/server/common/helpers/secure-context/secure-context.js

```diff
@@ -26,14 +28,13 @@ export const secureContext = {
             secureContext.context.addCACert(cert)
           })
 
+          server.logger.info('Custom secure context enabled')
+
           return secureContext
         }
-
-        // @ts-expect-error TS2769
-        server.decorate('server', 'secureContext', tls.createSecureContext())
-      } else {
-        server.logger.info('Custom secure context is disabled')
       }
+
+      server.decorate('server', 'getSecureContext', tls.createSecureContext)
```

Comment:
is this right? tls.createSecureContext still looks like its a function?

Reply:
yup its a function and called as a function when used. There is an example over in the backend template